### ISSUE TITLE
aws_elb_service_account errors for GovCloud region us-gov-west-1

### ISSUE THREAD

**Issue by mgresko** [terraform-providers/terraform-provider-aws#3373]
Terraform is erroring out on Govcloud region when trying to get the elb service account

### Terraform Version
Terraform v0.11.3
+ provider.aws v1.1.0
+ provider.template v1.0.0

### Affected Resource(s)
- aws_elb_service_account 

### Terraform Configuration Files
```hcl
data "aws_elb_service_account" "main" {}

data "aws_iam_policy_document" "logging" {
    statement {
        sid = "AllowLogsWritten"
        effect = "Allow"
        principals = {
            type = "AWS"
            identifiers = ["${data.aws_elb_service_account.main.arn}"]
        }
        actions = ["s3:PutObject"]
        resources = [
            "${aws_s3_bucket.logging.arn}/log/*"
        ]
    }
}
```

### Expected Behavior
The region should be found and resolve the account id 048591011584.  https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html

### Actual Behavior
Terraform error
```
Error: Error refreshing state: 1 error(s) occurred:

* data.aws_elb_service_account.main: 1 error(s) occurred:

* data.aws_elb_service_account.main: data.aws_elb_service_account.main: Unknown region ("us-gov-west-1")
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
GovCloud

**Comment by mgresko** [comment]
Nevermind... this is fixed in a newer version of provider.  Unfortunately, I am stuck at v1.1.0 until #2534 is fixed.  :(

**Comment by bflad** [comment]
@mgresko looks like this was actually fixed this week in #3315 and already merged into master. This will be released with v1.10.0 of the AWS provider, likely next week.

**Comment by mgresko** [comment]
@bflad if #2534 could make it in v1.10.0 then all would be well :)
